content/exception-filters.md

```diff
@@ -364,8 +364,8 @@ The first method is to inject the `HttpAdapter` reference when instantiating the
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
 
-  const { httpAdapter } = app.get(HttpAdapterHost);
-  app.useGlobalFilters(new AllExceptionsFilter(httpAdapter));
```

Comment:
This `AllExceptionsFilter` is not the same as the one up on line 300. This one `extends BaseExceptionFilter` which takes in an `HttpAdapter` instead of an `HttpAdapterHost`. The docs are correct

Reply:
Alright, that makes sense but it's not this clear in the docs, it almost looks like it's just a continuation.